tests/NuGet.Services.AzureSearch.FunctionalTests/Support/TokenizationData.cs

```diff
@@ -45,7 +50,7 @@ public static class TokenizationData
             { "HTML", new[] { "html"} },
             { "HTMLThing", new[] { "htmlthing" } },
             { "HTMLThingA", new[] { "htmlthinga", "htmlthing", "a" } },
-            { "HelloWorld𠈓Foo", new[] { "helloworld", "hello", "world𠈓foo" } },
+            { "HelloWorld𠈓Foo", new[] { "helloworld𠈓foo", "hello", "world𠈓foo" } },
```

Comment:
This test case was incorrect. The previous token was produced due to an incorrect truncate length of 10 characters.